package.json

```diff
@@ -1,22 +1,50 @@
 {
-  "name": "post-message-stream",
+  "name": "@metamask/post-message-stream",
   "version": "4.0.0",
   "description": "Sets up a duplex object stream over window.postMessage",
-  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "yarn build && yarn testOnly",
+    "testOnly": "jest --runInBand",
+    "test:watch": "jest --watch",
+    "prepublishOnly": "npm run build",
+    "lint": "eslint --ext .ts,.js ./src",
+    "build": "yarn run build:clean && yarn run build:compile && yarn run build:bundle",
+    "build:clean": "rm -rf ./dist && mkdir -p ./dist/esm && mkdir -p ./dist/cjs",
+    "build:compile": "yarn build:compile:esm && yarn build:compile:cjs",
+    "build:compile:esm": "tsc --module ES6 --outDir ./dist/esm",
+    "build:compile:cjs": "tsc --module UMD --outDir ./dist/cjs",
+    "build:bundle": "browserify --standalone PostMessageStream ./dist/cjs/index.js > ./dist/bundle.js"
   },
-  "author": "",
-  "license": "ISC",
+  "license": "MIT",
   "dependencies": {
-    "readable-stream": "2.3.3"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kumavis/post-message-stream.git"
-  },
-  "bugs": {
-    "url": "https://github.com/kumavis/post-message-stream/issues"
+    "@types/readable-stream": "^2.3.9",
+    "readable-stream": "^3.0.0"
   },
-  "homepage": "https://github.com/kumavis/post-message-stream#readme"
+  "devDependencies": {
+    "@jest-runner/electron": "^3.0.0",
+    "@metamask/eslint-config": "^3.2.0",
+    "@types/jest": "^26.0.13",
+    "@types/node": "^14.11.2",
+    "@typescript-eslint/eslint-plugin": "^4.1.1",
+    "@typescript-eslint/parser": "^4.1.1",
+    "browserify": "^16.5.2",
+    "electron": "^10.1.2",
+    "eslint": "^7.9.0",
+    "eslint-plugin-import": "^2.22.0",
+    "jest": "^26.4.2",
+    "merge": "^1.2.1",
+    "ts-jest": "^26.3.0",
+    "ts-node": "^9.0.0",
```

Comment:
We should be able to remove this from the list, I don't think we're using it anywhere